launch/launch/frontend/parser.py

```diff
@@ -14,15 +15,16 @@
 
 """Module for Parser class and parsing methods."""
 
-import io
-import os
+import os.path
 from typing import Any
 from typing import List
 from typing import Optional
 from typing import Text
+from typing import TextIO
 from typing import Tuple
 from typing import Type
 from typing import Union
+from typing import TYPE_CHECKING
```

Comment:
flake8 isn't passing in some of the platforms because this is not correctly ordered (and because of other unrelated errors).
Can you reorder? Thanks!

Reply:
Done. I didn't see this because https://github.com/ament/ament_lint/issues/254